buildspec.yml

```diff
@@ -0,0 +1,75 @@
+version: 0.2
+
+env:
+  variables:
+    FRAMEWORK_VERSION: "1.3.0"
+    ACCOUNT: "142577830533"
+    AWS_DEFAULT_REGION: "us-west-2"
+    ECR_REPO: "sagemaker-framework-pull-request-integ"
```

Comment:
+1 to separate repo for this. not sure about the name. also we should add this repo to the Dev/SandboxEcrStacks

Reply:
Do we have to separate the repo? Since images will be removed finally, I feel like one repo is good. We just distinguish images by tags with framework name in it.

Reply:
no, i meant using one pr image repo like you have done here is a good idea. multiple projects can share this. maybe a shorter name would be good, like sagemaker-test?

Reply:
Sure, updated to sagemaker-test